### DISCUSSION TITLE
Why I added a login delay

### DISCUSSION THREAD

**Discussion by EthanLipnik** [EthanLipnik/OpenSesame#11] [Q&A]
Much like how vacuums are designed to be loud and many services have fake loading bars, OpenSesame uses a short login delay so the user feels how secure it is. CryptoKit is so incredibly fast and low level that this would load instantly (much like iCloud Keychain), but this gives it a more solid feel.

**Should we keep this or should I remove it for instant loading?**

**Comment by dancojocaru2000** [comment]
Maybe add a somewhat hidden way to turn it off for people who know it's secure and wanna skip the inconvenient waiting.
